pyramid/tests/test_authentication.py

```diff
@@ -758,6 +767,12 @@ def test_identify_cookie_timed_out(self):
         result = helper.identify(request)
         self.assertEqual(result, None)
 
+    def test_identify_cookie_str_timeout(self):
+        helper = self._makeOne('secret', timeout='1')
```

Comment:
If you just want to test that `timeout` can be a `str`, then do a try/except block with a self.assertTrue(false) in the except block. So if it raises a ValueError then the test fails.